lib/board.ex

```diff
@@ -1,57 +1,47 @@
 defmodule Board do
 
-  @markers ["X","O"]
-
-  def create(board_size) do
-    {:x, [["1","2","3"],["4","5","6"],["7","8","9"]], :first_turn}
+  def create do
+    [["1","2","3"],["4","5","6"],["7","8","9"]]
   end
 
-  def place_marker(selected_tile, game_status) do
-    marker = elem(game_status, 0)
-    elem(game_status, 1)
-    |> check_tile_not_taken(selected_tile, marker)
-  end
+  def place_marker(selected_tile, board, player_symbol) do
+    marker_string = Markers.get_player_marker(player_symbol)
+    index_selection = zero_index_selection(selected_tile)
+    row_size = length(board)
 
-  def check_tile_not_taken(board, selected_tile, marker) do
-     selection_value = get_value_of_selected_tile(zero_index_selection(selected_tile), board)
-     Enum.member?(@markers,selection_value)
-     |> create_next_game_status(board, marker, selected_tile)
+    do_replace_marker_in_board(board, index_selection, marker_string, row_size)
```

Comment:
I would try to avoid using types in the variable names `marker_string`. I would encourage you to think of names like `player_mark` or something similar. This is because types can change (and then you would need to remember to change the variable name).